src/nav/design.yml

```diff
@@ -16,3 +16,5 @@
           title: 26px
         - id: /design/icons/wordmarks
           title: Wordmarks
+    - title: Color
+      id: https://garden.zendesk.com/css-components/utilities/color/
```

Comment:
Although marginally less attractive, let's point this url to https://zendeskgarden.github.io/react-components/theming/#!/PALETTE. In contrast with the `css-components` page, the hex values displayed are computed directly from the theme palette and can be counted on for source accuracy.